pom.xml

```diff
@@ -319,7 +315,7 @@
     <commons.jira.pid>12310469</commons.jira.pid>
     <commons.pool.version>2.12.0</commons.pool.version>
     <!-- See DBCP-445 and DBCP-454 -->
-    <commons.osgi.import>javax.transaction;version="1.1.0",javax.transaction.xa;version="1.1.0";partial=true;mandatory:=partial,*</commons.osgi.import>
+    <commons.osgi.import>jakarta.transaction;version="2.0.1",javax.transaction.xa;version="1.1.0";partial=true;mandatory:=partial,*</commons.osgi.import>
```

Comment:
I am not sure how this osgi stuff works, so maybe there is something more to change here...